src/gettingstarted/own-code/routes-configuration.md

```diff
@@ -0,0 +1,57 @@
+# Import your own code
+
+## Configure routes
+
+The final configuration file you will need to modify in your repository is the `.platform/routes.yaml` file, which describes how an incoming HTTP request is going to be processed by Platform.sh.
+
+```.
+├── .platform
+│   ├── routes.yaml
+│   └── services.yaml
+├── .platform.app.yaml
+└── < application code >
+```
+
+A minimal configuration `.platform/routes.yaml` for all languages will look very similar:
+
+{% codetabs name="PHP", type="php", url="https://raw.githubusercontent.com/platformsh/template-php/master/.platform/routes.yaml" -%}
+
+{%- language name="Go", type="js", url="https://raw.githubusercontent.com/platformsh/template-golang/master/.platform/routes.yaml" -%}
+
+{%- language name="Python", type="py", url="https://raw.githubusercontent.com/platformsh/template-python3/master/.platform/routes.yaml" -%}
+
+{% language name="Node.js", type="js", url="https://raw.githubusercontent.com/platformsh/platformsh-example-nodejs/master/.platform/routes.yaml" -%}
+
+{%- endcodetabs %}
+
+Configuring the routes can be done using either an absolute URL or a URL template as shown in the examples above that have the form `http://www.{default}`, where `{default}` will be substituted by either your configured domain or those automatically generated by Platform.sh.
+
+If you set up a domain of `example.com`, the route configuration `http://www.{default}` will be resolved to `http://www.example.com/`. Your production (`master`) environment's routes will be configured according to these rules, but so will each development environment that you activate.
+
+Each route can then be configured with the following properties:
+
+  * `type`:
+      * `upstream`: serves the application. Takes the form `upstream: <application name>:http`, using the application `name` set in your `.platform.app.yaml`.
+      * `redirect`: configures redirects from `http://{default}` to your application.
+  * `cache`: controls [caching behavior of the route](/configuration/routes/cache.md).
+  * `ssi`: controls whether [Server Side Includes](/configuration/routes/ssi.md) are enabled.
```

Comment:
Honestly we can probably remove the `ssi` entry here; I have never seen anyone use it, ever.  I am not even sure what it does in practice. :smile: